Use the elemental-rootfs feature

frelon opened the pull request:
The new elemental-rootfs and elemental-sysroot features are drop-in replacements for immutable-rootfs feature.

kkaempf reviewed (changes requested):
I'd rather see it split into two commits (or even PRs) ?
* the "elemental-rootfs" feature
* the rename from ".../cos/..." to ".../elemental/..."

frelon committed:
Use the elemental-rootfs feature

The new elemental-rootfs and elemental-sysroot features are drop-in
replacements for immutable-rootfs feature.

Signed-off-by: Fredrik Lönnegren <fredrik.lonnegren@suse.com>

frelon committed:
Migrate from /run/cos to /run/elemental

Signed-off-by: Fredrik Lönnegren <fredrik.lonnegren@suse.com>

kkaempf reviewed (approved):
Approving.
Next time, please create separate PRs for such cases
